src/app/judge-edit-dialog/judge-edit-dialog.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA} from '@angular/material';
+import * as _ from 'lodash';
+
+@Component({
+  selector: 'app-judge-edit-dialog',
+  templateUrl: './judge-edit-dialog.component.html',
+  styleUrls: ['./judge-edit-dialog.component.scss']
+})
+export class JudgeEditDialogComponent implements OnInit {
+  judgeNameOK = true;
+  judgeName: string;
+  test: any;
```

Comment:
test? Also, if it's a `Judge` maybe we need to create a `Judge.ts` and populate full-blown `Judge` objects in the Judge Service so everywhere is dealing with something typed